cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -54,22 +54,24 @@ func TestAddFlags(t *testing.T) {
 		"--audit-log-maxbackup=12",
 		"--audit-log-maxsize=13",
 		"--audit-log-path=/var/log",
-		"--audit-log-mode=blocking",
+		"--audit-log-mode=batch",
 		"--audit-log-batch-buffer-size=46",
 		"--audit-log-batch-max-size=47",
 		"--audit-log-batch-max-wait=48s",
 		"--audit-log-batch-throttle-enable=true",
 		"--audit-log-batch-throttle-qps=49.5",
 		"--audit-log-batch-throttle-burst=50",
+		"--audit-log-batch-max-routines=60",
 		"--audit-policy-file=/policy",
 		"--audit-webhook-config-file=/webhook-config",
-		"--audit-webhook-mode=blocking",
+		"--audit-webhook-mode=batch",
```

Comment:
It doesn't matter what these values represent, test is purely to check that non-default values are plumbed

You can revert these

Reply:
ok